Sources/SRGDataProviderRequests/SRGDataProvider+RadioRequests.m

```diff
@@ -159,4 +159,10 @@ - (NSURLRequest *)requestRadioCurrentSongForVendor:(SRGVendor)vendor
     return [self URLRequestForResourcePath:resourcePath withQueryItems:@[ [NSURLQueryItem queryItemWithName:@"onlyCurrentSong" value:@"true"] ]];
 }
 
+- (NSURLRequest *)requestAllRadioShowsForVendor:(SRGVendor)vendor
+{
+    NSString *resourcePath = [NSString stringWithFormat:@"integrationlayer/2.0/%@/showList/radio/alphabetical", SRGPathComponentForVendor(vendor)];
+    return [self URLRequestForResourcePath:resourcePath withQueryItems:@[ [NSURLQueryItem queryItemWithName:@"pageSize" value:@"unlimited"] ]];
```

Comment:
Since the request supports pagination I would suggest implementing pagination as well. If the whole list is desired for some reason (but this might take longer to load) the page size can be set to `SRGDataProviderUnlimitedPageSize`.